src/components/shared/TractionFieldGroup.vue

```diff
@@ -5,7 +5,13 @@
   - label: The label to apply
   - description: Longer text providing more help
   - component: the type of component to render (if not handled in the slot)
-  - componentProps: The configuration for the component
```

Comment:
This documentation is much better but I am still struggling to understand it. e.g. what is the traction field group for? You mention componentProps and what they are for but I am still slightly confused about what it means or maybe it is just me!

Reply:
I have added some more documentation. Please check if that is any good.